frontend/src/features/properties/hooks/useParcelLayerData.tsx

```diff
@@ -152,28 +155,55 @@ const useParcelLayerData = ({
   nameSpace,
   agencyId,
 }: IUseParcelLayerDataProps) => {
+  // Get LTSA information
+  const [ltsa, setLtsa] = useState<ILTSAOrderModel | undefined>(undefined);
+  const api = useApi();
+  // make the call to the LTSA api
+  const getLTSAData = async (pid: string) => {
+    try {
+      const ltsaData = await api.getLTSA(pid);
+      setLtsa(ltsaData);
+      return ltsaData;
+    } catch (e) {
+      setLtsa(undefined);
+    }
+  };
```

Comment:
Agree with you that we don't really need the state. Could just do this here. Plus some other changes seen below.
```suggestion
const api = useApi();
```